sql/parser2/expr.go

```diff
@@ -234,6 +234,19 @@ func (NullVal) String() string {
 // QualifiedName is a dot separated list of names.
 type QualifiedName []string
 
+// Database returns the database portion of the name.
+func (n QualifiedName) Database() string {
+	if len(n) > 1 {
+		return n[0]
+	}
+	return ""
+}
+
+// Table returns the table portion of the name.
+func (n QualifiedName) Table() string {
+	return n[len(n)-1]
```

Comment:
Oops, forgot to reply to this. `QualifiedName` is `[]string`. This is selecting the last element of the slice.